changelogs/fragments/wrcwd_ansible.cfg.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+    - avoid using ansible.cfg in a world readable dir, readd missing docs
```

Comment:
Since we don't have a dedicated category for security fixes, please prepend ```**Security Fix** -``` to this:
```
bugfixes:
    - ** Security Fix ** - avoid using ansible.cfg in a world readable dir, readd missing docs
```